sdk/src/Services/DynamoDBv2/Custom/DataModel/ContextInternal.cs

```diff
@@ -463,7 +463,14 @@ private object FromDynamoDBEntry(SimplePropertyStorage propertyStorage, DynamoDB
             var targetType = propertyStorage.MemberType;
             
             if (conversion.HasConverter(targetType))
-                return conversion.ConvertFromEntry(targetType, entry);
+            {
+                var output = conversion.ConvertFromEntry(targetType, entry);
+                if (targetType == typeof(DateTime) && flatConfig.RetrieveDateTimeInUtc)
```

Comment:
Can a user register their own `DateTime` converter? Wonder if we should only do this if it's the built-in one vs. something custom.

Reply:
Thanks, I didn't realize a custom converter isn't getting this far.